hack/make-rules/test-integration.sh

```diff
@@ -87,22 +93,11 @@ trap cleanup EXIT
 # If a test case is specified, just run once with v1 API version and exit
 if [[ -n "${KUBE_TEST_ARGS}" ]]; then
   runTests v1
+  exit 0
 fi
 
-# Pass arguments that begin with "-" and move them to goflags.
-what_flags=()
-for arg in "$@"; do
-  if [[ "${arg}" == -* ]]; then
-    what_flags+=("${arg}")
-  fi
-done
-if [[ "${#what_flags[@]}" -eq 0 ]]; then
-  what_flags=''
-fi
-
-
 # Convert the CSV to an array of API versions to test
 IFS=';' read -a apiVersions <<< "${KUBE_TEST_API_VERSIONS}"
```

Comment:
You don't really need to convert to an array here, the `for` will honor `$IFS` as is:
```sh
IFS=';'
for apiVersion in ${KUBE_TEST_API_VERSIONS}; do
```